README.md

```diff
@@ -1,56 +1,78 @@
+<a href="https://www.twilio.com">
```

Comment:
`README` has evolved over time. Would you mind using this one https://github.com/TwilioDevEd/eta-notifications-rails/blob/master/README.md as reference for updating this one. Any suggestions are welcome.